pkg/base/config.go

```diff
@@ -257,12 +257,14 @@ var (
 	// Raft log entries that a leader will send to a follower without hearing
 	// responses.
 	//
-	// TODO(#90314): lower this limit to something close to max rates observed in
-	// healthy clusters. Currently this value is a conservatively large multiple
-	// of defaultRaftMaxSizePerMsg * defaultRaftMaxInflightMsgs, so that it
-	// doesn't cut off traffic until we know it's ok.
-	defaultRaftMaxInflightBytes = envutil.EnvOrDefaultBytes(
-		"COCKROACH_RAFT_MAX_INFLIGHT_BYTES", 256<<20 /* 256 MB */)
+	// This env variable is used only if RaftConfig.RaftMaxInflightBytes is unset.
```

Comment:
This seems circular, since `RaftMaxInflightBytes` *is* the final setting and is never 0 at the end. Did you mean to say:

> If set to a nonzero value, this env var overrides RaftMaxInflightBytes, which is otherwise initialized from the values for RaftMaxSizePerMsg and RaftMaxInflightMsgs (with a generous factor applied).

It's minor and feel free to ignore me, but I still think that this is not the right place to say what the default value coming out of the default settings would be. If someone changed `defaultRaftMaxSizePerMsg`, how would they know to update this comment?

Reply:
The circularity is generally true for this `Config/RaftConfig`, see `RaftConfig.SetDefaults`, e.g.:
https://github.com/cockroachdb/cockroach/blob/59e7cabb81ed06f7011dee8a76abc7a7edb5dfc7/pkg/base/config.go#L539-L547

The config options are initialized from env vars unless they are already initialized.

Made this comment less specific, and move the note on the default value down to `SetDefaults`.

There are similar comments reasoning about the default assuming other defaults take place, it's generally hard to keep these all up-to-date, but I think there is some value in having them written down and assuming that changes are rare and eventually consistent.